.github/workflows/ci-new.yml

```diff
@@ -0,0 +1,233 @@
+# Copyright 2023 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# We will remove ci.yml and rename this to ci.yml
+# after the new integration test implementation
+# has finished.
+name: 'ci-new'
+
+# This will also be updated once the new integration
+# test implementation has finished
+on:
+  # this pull_request here is just for me to
+  # test this workflow works before merging to main
+  # I will remove pull_request and only keep
+  # workflow_dispatch before merging this PR to main.
+  pull_request:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+env:
+  PROJECT_ID: 'jvs-i-e5'
+  WIF_PROVIDER: 'projects/851681819867/locations/global/workloadIdentityPools/github-automation/providers/jvs-ci-i'
+  WIF_SERVICE_ACCOUNT: 'github-automation-bot@gha-jvs-ci-i-31a91c.iam.gserviceaccount.com'
+  CONTAINER_REGISTRY: 'us-docker.pkg.dev/jvs-i-e5/ci-images'
+  DOCKER_TAG: '${{ github.sha }}'
+  IMAGE_NAME: 'jvsctl'
+
+  API_SERVICE_NAME: 'jvs-api-4527'
+  CERT_ROTATOR_SERVICE_NAME: 'jvs-cert-rotator-e04a'
+  PUBLIC_KEY_SERVICE_NAME: 'jvs-public-key-f22e'
+  UI_SERVICE_NAME: 'jvs-ui-8ab7'
+  SERVICES_URL_PREFIX: 'https://'
+  SERVICES_URL_POSTFIX: '2nhpyabgtq-uc.a.run.app'
```

Comment:
Remove?

Reply:
I removed the `SERVICES_URL_PREFIX`, but I think `SERVICES_URL_POSTFIX` is necessary as in integ test, we will need this to put together the whole service url with tag. Something like `tag---api-service-postfix:443`.